tools/monitoring/main.go

```diff
@@ -25,15 +25,24 @@ import (
 	"os"
 
 	"github.com/knative/test-infra/tools/monitoring/config"
+	"github.com/knative/test-infra/tools/monitoring/mail"
 	"github.com/knative/test-infra/tools/monitoring/mysql"
 )
 
-var dbConfig mysql.DBConfig
+var (
+	dbConfig   mysql.DBConfig
+	mailConfig *mail.Config
+
+	alertEmailRecipients = []string{"knative-productivity-oncall@googlegroups.com"}
+)
 
 const (
-	yamlURL              = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
-	dbUserSecretFile     = "/secrets/cloudsql/monitoringdb/username"
-	dbPasswordSecretFile = "/secrets/cloudsql/monitoringdb/password"
+	yamlURL = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
+
+	dbUserSecretFile        = "/secrets/cloudsql/monitoringdb/username"
```

Comment:
It might be better make these configurable via flags

Reply:
Done